### ISSUE TITLE
More careportal integration

### ISSUE THREAD

**Issue by bmozisek** [timomer/HAPP#42]
This is totally a wish list item, but I wanted to ask anyway.  Would it be possible to add additional items  to be pushed up to nightscout careportal.  Since you are already pushing the temp rates, boluses and carbs, this might be an easy addition.  Specifically, I'm looking for Exercise, Notes, Pump Site Change, Insulin Cartridge Change, CGM Sensor Insert, CGM Sensor Start.

**Comment by timomer** [comment]
Yep, pushing should not be difficult with NSClient, capturing the data in a nice way will need some work and UI redesign.

**Comment by gempickfordwaugh** [comment]
I'm guessing pulling from NS is harder than pushing? It's easier to add carbs/bolus from pebble careportal app than having to get the phone out to open HAPP. I'm guessing it'd already be set up if it was easy so I'm probably asking for the impossible - feel free to say b*gger off!  What is already there is much appreciated!

**Comment by timomer** [comment]
To be honest with NSClient both should not be difficult at all. Its more the UI design and setting the time to one side. I have been having a good think about UI in the last few days and have a few ideas. Not top of the list but this is deff something I would like to add
